build/pom.xml

```diff
@@ -81,8 +81,8 @@
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
         <!-- The version of spring-boot for 'spring-boot-dependencies' and 'spring-boot-maven-plugin' -->
-        <spring-boot.version>2.5.15</spring-boot.version>
-        <spring-framework.version>5.3.27</spring-framework.version>
+        <spring-boot.version>2.7.18</spring-boot.version>
```

Comment:
The Spring dependencies for the client and server have been version-separated. The scope of the Spring version on the client side is set to "Provided," and it is maintained by the business side, allowing for the use of lower versions to ensure compatibility with more development frameworks. The server, seata-server, is independently deployed and has been upgraded to a higher version to address security vulnerabilities.

Reply:
> The Spring dependencies for the client and server have been version-separated. The scope of the Spring version on the client side is set to "Provided," and it is maintained by the business side, allowing for the use of lower versions to ensure compatibility with more development frameworks. The server, seata-server, is independently deployed and has been upgraded to a higher version to address security vulnerabilities.

The Spring API used by the Seata client will not become incompatible due to this version upgrade.